main.go

```diff
@@ -1,24 +1,55 @@
 package main
 
 import (
-    "fmt"
-    "os"
-    //"encoding/json"
-    //"io/ioutil"
-    //"strconv"
+	"flag"
+	"fmt"
+	"os"
+)
+
+var (
+	out = flag.String("out", "stdout", `Write the cost estimation to a given file path. If set to 'stdout',
+the output will be shown in the command line.`)
+
+	arityMismatch = 1
```

Comment:
`arityMismatch` and `fileErr` are actually constants, so

```
const (
    arityMismatch = 1
    fileErr       = 2
)
```
   
is better.

Another thought: Likely, the exit code doesn't need to be detailed (`os.Exit(1)` is good enough). Leaving this to you.